git_machete/client.py

```diff
@@ -1383,34 +1383,41 @@ def is_excluded_reflog_subject(sha_: str, gs_: str) -> bool:
         if not url_for_remote:
             raise MacheteException(fmt('No remotes defined for this repository (see `git remote`)'))
 
-        optional_org_name_for_github_remote: Dict[str, Optional[Tuple[str, str]]] = {
-            remote: get_parsed_github_remote_url(url) for remote, url in url_for_remote.items()}
-        org_name_for_github_remote: Dict[str, Tuple[str, str]] = {remote: org_name for remote, org_name in
-                                                                  optional_org_name_for_github_remote.items() if
-                                                                  org_name}
-        if not org_name_for_github_remote:
-            raise MacheteException(
-                fmt('Remotes are defined for this repository, but none of them '
-                    'corresponds to GitHub (see `git remote -v` for details)'))
-
-        org: str = ''
-        repo: str = ''
-        if len(org_name_for_github_remote) == 1:
-            org, repo = list(org_name_for_github_remote.values())[0]
-        elif len(org_name_for_github_remote) > 1:
-            if 'origin' in org_name_for_github_remote:
-                org, repo = org_name_for_github_remote['origin']
-            else:
+        remote: str
+        org: str
+        repo: str
+        remote, org, repo = self.get_remote_name_and_organization_and_repository_name_of_remote()
+        if not(remote and org and repo):
+            optional_org_name_for_github_remote: Dict[str, Optional[RemoteAndOrganizationAndRepository]] = {
+                remote: get_parsed_github_remote_url(url, remote) for remote, url in url_for_remote.items()}
+            org_name_for_github_remote: Dict[str, RemoteAndOrganizationAndRepository] = {remote: org_name for remote, org_name in
+                                                                                         optional_org_name_for_github_remote.items() if
+                                                                                         org_name}
+            if not org_name_for_github_remote:
                 raise MacheteException(
-                    f'Multiple non-origin remotes correspond to GitHub in this repository: '
-                    f'{", ".join(org_name_for_github_remote.keys())}, aborting')
+                    fmt('Remotes are defined for this repository, but none of them '
+                        'corresponds to GitHub (see `git remote -v` for details)'))
+            if len(org_name_for_github_remote) == 1:
+                _, org, repo = list(org_name_for_github_remote.values())[0]
+            elif len(org_name_for_github_remote) > 1:
+                if 'origin' in org_name_for_github_remote:
+                    _, org, repo = org_name_for_github_remote['origin']
+                else:
+                    raise MacheteException(
+                        f'Multiple non-origin remotes correspond to GitHub in this repository: '
```

Comment:
Here also the info about `machete.github.*` keys should be displayed to the user

Reply:
Okay... but isn't that one of the cases where the user might want to fall back to `machete.github.*` config keys? 🤔 as in, they see that _Multiple non-origin remotes correspond to GitHub in this repository_... but how to the pick up the right GitHub repo? I'd say that's a perfect moment to inform about this workaround